### PR TITLE
[sass/cs] Czech translation for Sass

### PR DESCRIPTION
**Pull request by MichalMartinek**
Czech translation for Sass

### CODE REVIEW

File: cs-cz/sass.html.markdown [hunk 1]
```diff
@@ -0,0 +1,439 @@
+---
+language: sass
+filename: learnsass.scss
+contributors:
+  - ["Laura Kyle", "https://github.com/LauraNK"]
+  - ["Sean Corrales", "https://github.com/droidenator"]
+translators:
+  - ["Michal Martinek", "https://github.com/MichalMartinek"]
+
```
Comment (adambard) [inline cs-cz/sass.html.markdown]: Could you add a `lang` line? I know it's a bit weird, but the term "language" gets a bit overloaded these days.

```
lang: cs-cz
```

File: cs-cz/sass.html.markdown [hunk 2]
```diff
@@ -0,0 +1,439 @@
+---
+language: sass
+filename: learnsass.scss
```
Comment (adambard) [inline cs-cz/sass.html.markdown]: Could you also change the filename to "learnsass-cz.scss" or some such thing? Otherwise there can be collisions.

**Comment by MichalMartinek** [comment]
Sorry for my late reply. It should be okay now.

**Comment by adambard** [comment]
looks good, thanks!

**Pull request merged**